<launch>
	<arg name="fusion_frame" 	default="/base_footprint"/>
	<arg name="cloud_in" 		default="/realsense2/depth/color/points"/>
 
	<node name="pcl_fusion_node_color" pkg="point_cloud_stitching_color" type="pcl_stitch_node_color" output="screen">
		<remap from="~input_point_cloud" to="$(arg cloud_in)" />
		<param name="fusion_frame" 		value="$(arg fusion_frame)"/>
		<rosparam param="bounding_box">[-1.0,1.0,-1.0,1.0,-1.0,1.0]</rosparam>
	</node>

</launch>